src/prpy/planning/ompl.py

```diff
@@ -163,10 +164,40 @@ def _Plan(self, robot, goal=None, tsrchains=None, timeout=30., shortcut_timeout=
         return traj
 
 
-class RRTConnect(OMPLPlanner):
-    def __init__(self):
-        OMPLPlanner.__init__(self, algorithm='RRTConnect')
+class OMPLRangedPlanner(OMPLPlanner):
+    """ Construct an OMPL planner with a default 'range' parameter set.
+
+    This class wraps OMPLPlanner by setting the default 'range' parameter,
+    which defines the length of an extension in algorithms like RRT-Connect, to
+    include an integer number 'multipler' of state validity checks.
+
+    @param algorithm name of the OMPL planner to create
+    @param robot_checker_factory robot collision checker factory
+    @param multiplier number of state validity checks per extension
+    """
+    def __init__(self, algorithm='RRTConnect', robot_checker_factory=None,
+            multiplier=1):
+        if multiplier < 1 or not isinstance(multiplier, int):
```

Comment:
Also, doesn't this mean that a `__lt__` or `__cmp__` override on multiplier could actually succeed here even if it is not an `int`?